locales/hi_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -83,15 +83,15 @@ msgctxt "noresults"
 msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
-msgstr ""
+msgstr "अगर आपको लगता है कि इस खोज के लिए परिणाम होने चाहिए, तो %sइधर दबाएँ%s"
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo!%s"
-msgstr ""
+msgstr "%sडकडकगो फैलाने में मदद करें!%s"
```

Comment:
`Duckduckgo` name will not be translated as it is a Trademark. Please leave it as it is.    
Do not do `डकडकगो`.